x-pack/legacy/plugins/apm/public/components/app/ServiceMap/Cytoscape.tsx

```diff
@@ -169,14 +172,22 @@ export function Cytoscape({
       }
     };
     const layoutstopHandler: cytoscape.EventHandler = event => {
-      event.cy.animate({
-        ...animationOptions,
-        center: {
-          eles: serviceName
-            ? event.cy.getElementById(serviceName)
-            : event.cy.collection()
+      setTimeout(() => {
```

Comment:
Don't you have to clean up this timeout on unmount?

Reply:
I think you're right. I'll update it. I'm not sure if it will ever need cleanup in practice, but better to be safe here.